examples/NuPIC Walkthrough.ipynb

```diff
@@ -881,28 +881,6 @@
      ]
     },
     {
-     "cell_type": "code",
-     "collapsed": false,
```

Comment:
This cell won't run because neither `ModelFactory` nor `MODEL_PARAMS` is defined so it will raise NameError and stop the "run all cells" command. And this line is already in the "Loading Data" section thus unnecessary here except that this cell is teaching how to fail the ipython notebook which I think is highly unlikely.